sdk/src/main/java/io/verloop/sdk/ui/VerloopActivity.kt

```diff
@@ -82,14 +82,12 @@ class VerloopActivity : AppCompatActivity() {
     }
 
     override fun onResume() {
-        Log.d(TAG, "onResume")
```

Comment:
There are other Log.d in this activity, can we put all of them in to our new logListener for debug flag?